sigs.yaml

```diff
@@ -3697,6 +3697,74 @@ workinggroups:
     liaison:
       github: saschagrunert
       name: Sascha Grunert
+- dir: wg-node-lifecycle
+  name: Node Lifecycle
+  mission_statement: >
+    Explore and improve node and pod lifecycle in Kubernetes. This should result in
+    better node drain/maintenance support and better pod disruption/termination. It
+    should also improve node and pod autoscaling, better application migration and
+    availability, load balancing, de/scheduling, node shutdown, and cloud provider
+    and third-party integrations.
+
+  charter_link: charter.md
+  stakeholder_sigs:
+  - Apps
+  - Autoscaling
+  - CLI
+  - Cloud Provider
+  - Cluster Lifecycle
+  - Network
+  - Node
+  - Scheduling
+  - Storage
+  label: node-lifecycle
+  leadership:
+    chairs:
+    - github: JoelSpeed
+      name: Joel Speed
+      company: Red Hat
+      email: Joel.speed@hotmail.co.uk
+    - github: atiratree
+      name: Filip Křepinský
+      company: Red Hat
+      email: atiratree@gmail.com
+    - github: fabriziopandini
+      name: Fabrizio Pandini
+      company: VMware
+      email: fabrizio.pandini@gmail.com
+    - github: haircommander
+      name: Peter Hunt
+      company: Red Hat
+      email: pehunt@redhat.com
+    - github: humblec
+      name: Humble Chirammal
+      company: VMware
```

Comment:
@atiratree can you please correct above line to be "VMware by Broadcom" ? may be line 3733 too .. cc @fabriziopandini

Reply:
Ack, updated.

Reply:
Unfortunately, this does not pass validation when matching with the older entries. You will have to update all entries in bulk in a separate PR.